docs/machine-learning/computational-bayesian-stats.md

```diff
@@ -53,7 +53,7 @@ class Normal:
 Now, I also stated that the normal distribution has a math function
 that we can use to allocate credibility points to the number line.
 This function also has a name,
-called a "probability distribution function", or the "PDF".
+called a "probability density function", or the "PDF".
```

Comment:
Ah yes, I saw this one, but didn't find the bandwidth to correct it in time. Thank you!